frontend/integration-tests-cypress/cypress-e2e/cypress/tests/monitoring_spec.ts

```diff
@@ -0,0 +1,89 @@
+import { testName, submitButton, errorMessage } from '../support/index';
+
+const testDetailsPage = (pageTitle: string, sectionHeading: string, testLabel = true) => {
+  cy.getTestElement('resource-title', 18000).contains(pageTitle); // long timeout for detail page to load
```

Comment:
I'm inclined to extract the `1800` into a `timeouts` file, prob `enum` it as something like:

```javascript
timeouts {
  pageLoad: 18000
  menuOpen: 500
  // etc
}
```
I'm guessing we have reasonable values scattered around our other tests.

Reply:
added